htdocs/projet/ajax/projects.php

```diff
@@ -62,7 +62,7 @@
 
 require_once DOL_DOCUMENT_ROOT.'/core/class/html.formprojet.class.php';
 
-top_httphead('application/json');
+top_httphead();
```

Comment:
Last line of file is print dol_json_encode, so content type should be json.

Reply:
You're right but the function is never executed on mode 'gettask' since $formproject->selectTasks already prints the list and then return...

Reply:
So it's better to convert the gettask to json in this case?

Reply:
I see. Well we have 2 diffrrent types of return so yes we must add a condition and return 2 different types of content type depending on condition on gettasks.
Can you update pr like this ?

Reply:
Yes, i will update the PR